README.md

```diff
@@ -1,22 +1,29 @@
 # BreweryDB API Wrapper in Golang
 
+[![Build Status](https://travis-ci.org/hobbeswalsh/brewerydb.svg?branch=master)](https://travis-ci.org/hobbeswalsh/brewerydb)
+
 ## To Use:
 
-```
+```go
+    package main
+
     import (
     	"github.com/michaelporter/brewerydb"
     )
+    
+    c := brewerydb.NewClient("myApiKey")
+
+    // I want a random beer.
+    beer, err := c.RandomBeer()
 
-    func getBeers() {
-      brewerydb.SetApiKey("your-api-key")
-      
-      queryParams := map[string]string{
-      	'abv': '6',
-      }
-      res, err := brewerydb.Get("/beers", queryParams)
-    }
+    // I want a list of breweries near some zip code
+    lq = brewerydb.LocationQuery{}
```

Comment:
oops! moving from other languages to Golang kinda sucks for variable assignment :)